$NetBSD: patch-ab,v 1.2 2011/10/01 21:42:16 dholland Exp $

- use modern C
- use the right include files for netbsd with wscons

--- kbdio/kbdio.y.orig	Mon Dec  4 13:57:07 2000
+++ kbdio/kbdio.y
@@ -40,7 +40,13 @@
 #include <math.h>
 #include <sys/fcntl.h>
 #include <machine/cpufunc.h>
+#include <unistd.h>
+#if WSCONS
+#include <sys/ioctl.h>
+#include <dev/wscons/wsdisplay_usl_io.h>
+#else
 #include <machine/pcvt_ioctl.h>
+#endif
 
 #ifdef __NetBSD__
 #include <machine/pio.h>
